app/assets/javascripts/application_non_webpack.js

```diff
@@ -1,5 +1,3 @@
 // All webpack assets in development will be loaded via webpack dev server
 
 // turbolinks comes from npm and is listed in webpack.client.base.config.js
-
-//= require rails_startup
```

Comment:
This file is no longer needed, as we do the styling in react, rather than jQuery. Old file is immediately below.